tests/conftest.py

```diff
@@ -87,7 +87,10 @@ def minimum_algorithm_musplit() -> dict:
             "multiscale_count": 4,
             "predict_logvar": "pixelwise",
         },
-        "likelihood": {"type": "GaussianLikelihoodConfig", "color_channels": 2}, # TODO this is incorrect
+        "likelihood": {
+            "type": "GaussianLikelihoodConfig",
+            "color_channels": 2,
+        },  # TODO this is incorrect
```

Comment:
In case you wrote that `TODO`, could you add more details as why this is incorrect? Otherwise for anyone else it will be more complicated to figure it out.

Reply:
It was me, I'll update that :)